ci_jobs_trigger/libs/openshift_ci/re_trigger/re_trigger.py

```diff
@@ -184,6 +184,6 @@ def get_url_content(self, **kwargs):
     def generate_slack_msg_prefix(self):
         return f"""
 Job: {self.job_name}
-Build ID: {self.build_id}
+Build ID: <{PROW_LOGS_URL_PREFIX}/{self.job_name}/{self.build_id}|{self.build_id}>
```

Comment:
Maybe change `Build ID` to `Build logs` ?

Reply:
@myakove I can do that, it will still display the build id, it will just be a hyperlink to the prow build. 

Would you rather a separate line for the build logs and we can leave the "Build ID" line the way it is currently?

Reply:
I see now, then leave it as is.